components/style/build_gecko.rs

```diff
@@ -612,6 +612,8 @@ mod bindings {
                 .borrowed_type(ty)
                 .zero_size_type(ty, &structs_types);
         }
+        builder = builder
+            .raw_line(format!("pub use gecko_bindings::structs::root::*;"));
```

Comment:
This basically kills the point of explicitly mentioning the types to include, so I'd rather not do this. Which libclang version are you using? Why do the types end up using `__uint32_t` instead of uint32_t, which bindgen knows how to translate to u32?